modules/classes.py

```diff
@@ -87,210 +92,222 @@ def __init__(
                 starty = self.end[1]
-            elif self.angle > 0 and self.angle < math.pi:
+            elif 0 < self.angle < math.pi:
                 start = 0
-                end = math.pi/2
+                end = math.pi / 2
                 startx = self.end[0]
                 starty = self.start[1]
-            elif self.angle < 0 and self.angle > -math.pi/2:
+            elif 0 > self.angle > -math.pi / 2:
                 start = -math.pi
-                end = -math.pi/2
+                end = -math.pi / 2
                 startx = self.end[0]
                 starty = self.start[1]
-            elif self.angle < 0 and self.angle > -math.pi:
-                start = math.pi/2
+            elif 0 > self.angle > -math.pi:
+                start = math.pi / 2
                 end = math.pi
                 startx = self.start[0]
                 starty = self.end[1]
-                
-            lin = np.linspace(start,end,num = 10)
-            r =  self.end[0]-self.start[0]
-            X = startx+np.cos(lin)*abs(r)
-            Y = starty+np.sin(lin)*abs(r)
-            out = [X,Y, self.thickness,self.material,_PLACE_[self.tag]]
+
+            lin = np.linspace(start, end, num=10)
+            r = self.end[0] - self.start[0]
+            X = startx + np.cos(lin) * abs(r)
+            Y = starty + np.sin(lin) * abs(r)
+            out = [X, Y, self.thickness, self.material, _PLACE_[self.tag]]
 
         return out
 
     def save_data(self):
-        return [self.start,self.end,self.thickness*1e3,self.material,_PLACE_[self.tag]]
+        return [self.start, self.end, self.thickness * 1e3, self.material, _PLACE_[self.tag]]
 
-    def calc_I_global(self, Ixx_c, Iyy_c, axis = 'x'):
-        ''' Calculate the moments relative to an axis. The axis argument is either passed as an string 'x' or 'y'(to indicate the Global Axis)
-            or an custom Vertical or Horizontal Axis as a dictionary
-            ie. axis = { 'axis' : 'x', 'offset' : 1.0} (This indicates an horizontal axis offset to the global axis positive 1 unit.)         '''
+    def calc_I_global(self, Ixx_c, Iyy_c, axis='x'):
+        """
+        Calculate the moments relative to an axis. The axis argument is either passed as an string 'x' or 'y'
+        (to indicate the Global Axis) or a custom Vertical or Horizontal Axis as a dictionary
+        i.e. axis = { 'axis' : 'x', 'offset' : 1.0}
+        This indicates a horizontal axis offset to the global axis positive 1 unit.
+        """
         if axis == 'x':
-            #Default Global axis for the prime forces
-            Ixx = Ixx_c + self.CoA[1]**2*self.area
+            # Default Global axis for the prime forces
+            Ixx = Ixx_c + self.CoA[1] ** 2 * self.area
             return Ixx
         elif axis == 'y':
-            Iyy = Iyy_c + self.CoA[0]**2*self.area
+            Iyy = Iyy_c + self.CoA[0] ** 2 * self.area
             return Iyy
-        elif type(axis) == dict:
+        elif axis is dict:
```

Comment:
Bruh .... 
```py
if type(axis) is dict: 
```
correct check, while
```py
axis is dict
```
introduced erroneous response. 
@gerelef

Reply:
From a manual check just now:

```python3
>>> x = {}
>>> type(x) == dict
True
>>> x is dict
False
>>> isinstance(x, dict)
True
```
I was fixing stuff fast and blundered; can you check where I changed type(x) == SOME_TYPE to x is SOME_TYPE?
I should've used isinstance, my fault.